src/axolotl/core/trainers/mixins/dist_parallel.py

```
@@ -0,0 +1,41 @@
+
+class DistParallelMixin(transformers.trainer.Trainer):
+    """
+    Trainer mixin to patch Accelerator for distributed parallel training
+    """
+
+    def create_accelerator_and_postprocess(self):
+        res = super().create_accelerator_and_postprocess()
+
+        if not is_package_version_ge("accelerate", "1.10.0"):
+            # pylint: disable=protected-access
+            if int(os.environ.get("WORLD_SIZE", "1")) > 1:
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)